sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/thriftserver/HiveThriftServer2Suites.scala

```diff
@@ -29,6 +27,8 @@ import scala.concurrent.{Await, Promise}
 import scala.sys.process.{Process, ProcessLogger}
 import scala.util.{Random, Try}
 
+import com.google.common.base.Charsets
```

Comment:
Could you use `com.google.common.base.Charsets.UTF_8`? So that after upgrading to Java 7, we can simply update the imports without touching the codes.